client/core/core.go

```diff
@@ -6282,6 +6321,13 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 // cached, no new requests will be made.
 func (c *Core) cacheRedemptionFeeSuggestion(t *trackedTrade) {
 	// Try to find any book that might have the fee.
```

Comment:
Would you mind moving this comment down near bestBookFeeSuggestion?  It's confusing here now.